File: Easy/Python/1929.-Concatenation-of-Array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n=len(nums);a=[0]*(2*n)
        for i in range(2*n):
            if i<n:
                a[i]=nums[i]
            else:
                a[i]=nums[i-n]
        return a

File: Easy/Java/1929.-Concatenation-of-Array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int l=nums.length;
        int ans[] = new int[l*2];
        for (int i=0;i<l;i++){
            ans[i]=nums[i];
        }
        for (int i=l;i<2*l;i++){
            ans[i]=nums[i-l];
        }
        return ans;
    }
}